crates/interfaces/Cargo.toml

```diff
@@ -0,0 +1,32 @@
+[package]
+name = "interfaces"
```

Comment:
Interfaces is a very broad name and can mean just about anything. This is our contract interface types we're talking about, so I'd prefer to name this crate something like `contract_abi_types` or maybe just `abi_types`